otherlibs/unix/getgr.c

```diff
@@ -43,31 +46,77 @@ static value alloc_group_entry(struct group *entry)
 
 CAMLprim value caml_unix_getgrnam(value name)
 {
-  struct group * entry;
+  value res;
+  struct group *resultp;
   if (! caml_string_is_c_safe(name)) caml_raise_not_found();
+
+#ifdef HAVE_GETGRNAM_R
+  long int initlen = sysconf(_SC_GETGR_R_SIZE_MAX);
```

Comment:
```suggestion
  long initlen = sysconf(_SC_GETGR_R_SIZE_MAX);
```
Did you paste this from an example in the spec?

Reply:
I did take some inspiration it, yes.